### PR TITLE
db/install.xml fix unit test on ubuntu server

### PR DESCRIPTION
**Pull request by james-voong**
Fix unit test failure on ubuntu servers

### CODE REVIEW

File: db/install.xml [hunk 1]
```diff
@@ -20,4 +20,4 @@
       </INDEXES>
     </TABLE>
   </TABLES>
-</XMLDB>
\ No newline at end of file
+</XMLDB>
```
Comment (roperto) [inline db/install.xml]: I am not sure why we need to add the newline (Moodle unfortunately creates this way)

**Comment by roperto** [comment]
Changing the `.sh` permissions is making CI fail as they use one of those scripts to initialise the environment.

File: db/install.xml (hunk 1 above)
Comment (james-voong) [reply db/install.xml]: This is breaking the unit test on ubuntu servers, adding the newline fixes the test.

**Comment by james-voong** [comment]
Hi, yes changing the .sh permissions is making the CI fail. I'm still trying to think up a solution. Perhaps edit the unit tests specifically for .sh files? Currently, having executable permissions on them will cause the unit test to fail.

**Comment by roperto** [comment]
Hi @james-voong -- where are the failing unit tests? How can I reproduce?

Moodle CI tests were green, but I am still in favour of removing the extra permissions from the PHP files

**Comment by james-voong** [comment]
Hello, try using totara10.

**Comment by james-voong** [comment]
The test that's failing due to permissions:

```
Testing suite totara_core_testsuite
Totara 10.9 (Build: 20180514.00)
PHP: 7.0.30, pgsql: 9.5.11, OS: Linux 4.4.0-112-generic x86_64
PHPUnit 5.7.27 by Sebastian Bergmann and contributors.

...............................................................  63 / 241 ( 26%)
............................................................... 126 / 241 ( 52%)
............................................................... 189 / 241 ( 78%)
.F..................................................            241 / 241 (100%)

Time: 39.49 seconds, Memory: 108.00MB

There was 1 failure:

1) totara_core_totara_testcase::test_file_bitmask
auth/saml2/tests/idp/simplesamlphp.sh is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/modules/statistics/bin/logcleaner.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/modules/statistics/bin/loganalyzer.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/modules/metarefresh/bin/metarefresh.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/modules/cron/bin/cron.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/modules/oauth/bin/demo.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/memcacheSync.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/importPdoMetadata.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/initMDSPdo.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/build-release.sh is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/pwgen.php is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/check-syntax.sh is not correctly bitmasked, it is using 775
auth/saml2/extlib/simplesamlphp/bin/ldapattrschemaparser.pl is not correctly bitmasked, it is using 775

/var/www/html/totara/core/tests/totara_test.php:63
/var/www/html/lib/phpunit/classes/base_testcase.php:600
/var/www/html/lib/phpunit/classes/advanced_testcase.php:68

To re-run:
 vendor/bin/phpunit --colors totara_core_totara_testcase totara/core/tests/totara_test.php

FAILURES!
Tests: 241, Assertions: 57793, Failures: 1.
```

**Comment by roperto** [comment]
Merged, thank you @james-voong for the fix

**Pull request merged**